libpod/define/container_inspect.go

```diff
@@ -30,6 +30,8 @@ type InspectContainerConfig struct {
 	StdinOnce bool `json:"StdinOnce"`
 	// Container environment variables
 	Env []string `json:"Env"`
+	// Unset container environment variables
+	UnsetEnv []string `json:"UnsetEnv"`
```

Comment:
Can you make this `OmitEmpty`?

Reply:
Also - where's this getting set? I don't see any changes to `container_inspect.go` in libpod

Reply:
I removed it, since we only want to see the actual Env in the container.  Unsetenv and UnsetenvAll have already done their thing.